internal/transport/http_util.go

```diff
@@ -130,6 +130,8 @@ type parsedHeaderData struct {
 	grpcErr        error
 	httpErr        error
 	contentTypeErr string
+
+	httpMethod string
```

Comment:
This can probably be moved to the set of fields under `Server side only fields` (line 110)

Reply:
Done.